src/index.ts

```diff
@@ -577,6 +577,28 @@ export class ERC725<Schema extends GenericSchema> {
     return leftPad(toHex(result), 64);
   }
 
+  /**
+   * Encode permissions into a hexadecimal string as defined by the LSP6 KeyManager Standard.
+   *
+   * @link https://github.com/lukso-network/LIPs/blob/main/LSPs/LSP-6-KeyManager.md LSP6 KeyManager Standard.
+   * @param permissions The permissions you want to specify to be included or excluded. Any ommitted permissions will default to false.
+   * @returns {*} The permissions encoded as a hexadecimal string as defined by the LSP6 Standard.
+   */
+  encodePermissions(permissions: {
+    CHANGEOWNER?: boolean;
+    CHANGEPERMISSIONS?: boolean;
+    ADDPERMISSIONS?: boolean;
+    SETDATA?: boolean;
+    CALL?: boolean;
+    STATICCALL?: boolean;
+    DELEGATECALL?: boolean;
+    DEPLOY?: boolean;
+    TRANSFERVALUE?: boolean;
+    SIGN?: boolean;
```

Comment:
maybe you can create an interface for this and use the same interface for the static and non static method?